index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <script src="challenge.js"></script>
+    <title>Challenge</title>
+  </head>
+  <body >
+        <header>
+          <h1>Adam Smith</h1>
+        </header>
+       <p id="bio">
+         Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <a href="https://www.linkedin.com"> tempor</a> tempor incididunt ut labore et dolore magna aliqua.  Ut si enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+       </p>
+       <h1>Education history</h1>
```

Comment:
- [ ] When giving a review, it is usually good to always includes an explanation ("Why") for each requested change so that the person whose code is being reviewed will understand why the change was necessary and why it should be implemented 😄